core/templates/components/state-editor/state-hints-editor/state-hints-editor.component.ts

```diff
@@ -13,231 +13,230 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive for the add and view hints section of the state
+ * @fileoverview Component for the add and view hints section of the state
  * editor.
  */
 
-import { DeleteHintModalComponent } from
-  // eslint-disable-next-line max-len
-  'pages/exploration-editor-page/editor-tab/templates/modal-templates/delete-hint-modal.component';
-import { DeleteLastHintModalComponent } from
-  // eslint-disable-next-line max-len
-  'pages/exploration-editor-page/editor-tab/templates/modal-templates/delete-last-hint-modal.component';
-
-require(
-  'components/common-layout-directives/common-elements/' +
-  'confirm-or-cancel-modal.controller.ts');
-require('components/state-directives/hint-editor/hint-editor.component.ts');
-require(
-  'components/state-directives/response-header/response-header.component.ts');
-require('domain/exploration/HintObjectFactory.ts');
-require('domain/utilities/url-interpolation.service.ts');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-editor.service.ts');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-hints.service.ts');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-interaction-id.service.ts');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-next-content-id-index.service');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-solution.service.ts');
-require('filters/format-rte-preview.filter.ts');
-require('services/alerts.service.ts');
-require('services/context.service.ts');
-require('services/editability.service.ts');
-require('services/generate-content-id.service.ts');
-require('services/contextual/window-dimensions.service.ts');
-require('services/external-save.service.ts');
-require('services/ngb-modal.service.ts');
-
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
```

Comment:
For this chunk of imports, let's move the `@angular` and `@ng-bootstrap` imports to the top.

Reply:
done